packages/blockchain-ui/src/components/elements/TutorialTile/TutorialTile.scss

```diff
@@ -0,0 +1,66 @@
+@import '@carbon/colors/scss/colors';
+@import '@carbon/layout/scss/_spacing';
+
+.tab-container {
+    display: flex;
+    flex-direction: column;
+    max-width: 70%;
+    background-color: white;
+    margin-bottom: $carbon--spacing-03;
```

Comment:
you shouldn't need to do `$carbon--spacing-03` or whatever, just `$spacing-03` should be fine

Reply:
Done!